enhancements/proposals/2549-scylladbcluster-backup-repair/README.md

```diff
@@ -0,0 +1,590 @@
+
+ScyllaDB Manager agent containers require credentials to the corresponding regional object storage.
+Users will be responsible for configuring the credentials for each datacenter by defining `scylladbclusters.scylla.scylladb.com/v1alpha1.spec.datacenters[].scyllaDBManagerAgent.volumes` and `scylladbclusters.scylla.scylladb.com/v1alpha1.spec.datacenters[].scyllaDBManagerAgent.volumeMounts`.
+
+### Test Plan
+
+The existing set of E2E tests related to ScyllaCluster integration with ScyllaDB Manager will be maintained in an unchanged form.
+The tests will cover correctness of ScyllaCluster's task spec conversion to ScyllaDBManagerTasks' specs and ScyllaDBManagerTasks' statuses to ScyllaClusters' task statuses.
+
+Conversion logic between ScyllaCluster's (scylla.scylladb.com/v1) in-manager state, reflected in its status, and ScyllaDBManagerClusterRegistration (scylla.scylladb.com/v1alpha1) will be covered by unit tests.
+Conversion logic between ScyllaCluster's (scylla.scylladb.com/v1) task specification and ScyllaDBManagerTasks (scylla.scylladb.com/v1alpha1) will be covered by unit tests.
+
+A set of new E2E tests will be introduced to test ScyllaDBCluster and ScyllaDBDatacenter integration, including:
+- Registering ScyllaDBDatacenters with global ScyllaDB Manager.
+- Registering ScyllaDBClusters with global ScyllaDB Manager.
+- Scheduling backup and repair tasks, defined by ScyllaDBManagerTask objects, with global ScyllaDB Manager.
+  ScyllaDBManagerTasks will be tested with references to ScyllaDBClusters and ScyllaDBDatacenters.
+  Tests for backup task scheduling will not cover the workaround for when AuthN is enabled [#2548][4].
+
+In order to set up E2E tests covering backup task scheduling for ScyllaDBClusters in multi-datacenter environments, the test-runner and related scripts will be extended to accept object storage bucket location and credentials for multiple datacenters.
+The relevant CI job definitions will also be adjusted.
+No changes are expected in the CI tooling.
+
+### Upgrade / Downgrade Strategy
+
+The new ScyllaDBManagerClusterRegistration (scylla.scylladb.com/v1alpha1) and ScyllaDBManagerTask (scylla.scylladb.com/v1alpha1) CRDs will have to be installed ahead of Scylla Operator upgrade. 
+Our documentation and release notes cover this requirement for both GitOps and Helm installation paths on the regular basis.
+
+As the existing manager-controller deployment is dropped with this proposal, users following the Helm installation path will have to upgrade their scylla-manager release ahead of Scylla Operator upgrade. 
+Users following the GitOps installation path will have to manually delete the manager-controller deployment from `scylla-manager` namespace, and perform the regular update of ScyllaDB Manager manifests ahead of Scylla Operator upgrade.
+Release notes should explicitly mention and highlight the manual steps required on the GitOps path. 
```

Comment:
Have you verified if Helm would remove the deployment upon upgrade? If it wouldn't we would have to mention it for Helm as well.